src/System.ValueTuple/tests/ExtensionsTests.cs

```diff
@@ -673,27 +672,27 @@ private struct IntHolder
         public override string ToString()
         {
             var builder = new StringBuilder();
-            if (x1 > 0) { builder.Append($"{x1}"); }
-            if (x2 > 0) { builder.Append($" {x2}"); }
-            if (x3 > 0) { builder.Append($" {x3}"); }
-            if (x4 > 0) { builder.Append($" {x4}"); }
-            if (x5 > 0) { builder.Append($" {x5}"); }
-            if (x6 > 0) { builder.Append($" {x6}"); }
-            if (x7 > 0) { builder.Append($" {x7}"); }
-            if (x8 > 0) { builder.Append($" {x8}"); }
-            if (x9 > 0) { builder.Append($" {x9}"); }
-            if (x10 > 0) { builder.Append($" {x10}"); }
-            if (x11 > 0) { builder.Append($" {x11}"); }
-            if (x12 > 0) { builder.Append($" {x12}"); }
-            if (x13 > 0) { builder.Append($" {x13}"); }
-            if (x14 > 0) { builder.Append($" {x14}"); }
-            if (x15 > 0) { builder.Append($" {x15}"); }
-            if (x16 > 0) { builder.Append($" {x16}"); }
-            if (x17 > 0) { builder.Append($" {x17}"); }
-            if (x18 > 0) { builder.Append($" {x18}"); }
-            if (x19 > 0) { builder.Append($" {x19}"); }
-            if (x20 > 0) { builder.Append($" {x20}"); }
-            if (x21 > 0) { builder.Append($" {x21}"); }
+            if (x1 > 0) { builder.Append(x1); }
+            if (x2 > 0) { builder.Append(" " + x2); }
+            if (x3 > 0) { builder.Append(" " + x3); }
+            if (x4 > 0) { builder.Append(" " + x4); }
+            if (x5 > 0) { builder.Append(" " + x5); }
+            if (x6 > 0) { builder.Append(" " + x6); }
+            if (x7 > 0) { builder.Append(" " + x7); }
+            if (x8 > 0) { builder.Append(" " + x8); }
+            if (x9 > 0) { builder.Append(" " + x9); }
+            if (x10 > 0) { builder.Append(" " + x10); }
+            if (x11 > 0) { builder.Append(" " + x11); }
+            if (x12 > 0) { builder.Append(" " + x12); }
+            if (x13 > 0) { builder.Append(" " + x13); }
+            if (x14 > 0) { builder.Append(" " + x14); }
+            if (x15 > 0) { builder.Append(" " + x15); }
+            if (x16 > 0) { builder.Append(" " + x16); }
+            if (x17 > 0) { builder.Append(" " + x17); }
+            if (x18 > 0) { builder.Append(" " + x18); }
+            if (x19 > 0) { builder.Append(" " + x19); }
+            if (x20 > 0) { builder.Append(" " + x20); }
+            if (x21 > 0) { builder.Append(" " + x21); }
```

Comment:
So, I made a mistake above. NetDevFx1 builds mscorlib using `D:\NetFXDev1\src\Tools\x86\managed\v4.5\Roslyn\csc2.exe`, not version 4.0 that I thought.

From discussion with Aleksey, we think this corresponds to an unfinished version of C# 6, because this is a managed compiler (meaning Roslyn and C# 6), but `csc2` is a compiler server that never shipped. 
That explains why it's able to use some C# 6 features (expression-bodied properties), but not others (string interpolation with `$"..."`).
So I will have to remove the `langver` constraint.